habitat_baselines/evaluate_ppo.py

```diff
@@ -38,12 +56,53 @@ def main():
         default="configs/tasks/pointnav.yaml",
         help="path to config yaml containing information about task",
     )
+    parser.add_argument(
+        "--video-option",
+        type=int,
+        default=0,
+        choices=[0, 1, 2, 3],
+        help="option for video generation. 0: no video; 1: save videos in video-dir;"
+        " 2: upload video to tensorboard; 3: both save in video-dir and upload to tensorboard",
+    )
+    parser.add_argument("--video-dir", type=str)
+    parser.add_argument("--tensorboard-dir", type=str, default="tb_eval")
+
     args = parser.parse_args()
+    assert (args.model_path is not None) != (
+        args.tracking_model_dir is not None
+    ), "Must specify a single model or a directory of models, but not both"
 
-    device = torch.device("cuda:{}".format(args.pth_gpu_id))
+    writer_kwargs = dict(
+        log_dir=args.tensorboard_dir, purge_step=0, flush_secs=30
+    )
+    with (tensorboard.SummaryWriter(**writer_kwargs)) as writer:
```

Comment:
Can you add an option to not specify a tensorboard writer also, if someone does not want to use that.